### PR TITLE
image 0.25 support

### PR DESCRIPTION
**Pull request by frederik-uni**

### CODE REVIEW

**Comment by conways-glider** [comment]
Hello! Thanks for this contribution! Just so you know, I tend to do the version bumps myself just for extra verification. If you remove that change and tests pass, I'll go ahead and merge!

File: Cargo.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [package]
 name = "identicon-rs"
-version = "5.0.0"
+version = "5.0.1"
```
Comment (conways-glider) [inline Cargo.toml]: If you drop this change, it looks ready to merge. For the moment, version bumps are intentionally their own commits for now that I follow up with. I want to automate cutting versions some more though through some CI/CD. I just gotta figure out the templating or automatic version bumps.

**Review by conways-glider — CHANGES_REQUESTED** [review]
Just one minor nit- otherwise this looks great! Thanks for the contribution!

File: Cargo.toml (hunk 1 above)
Comment (frederik-uni) [reply Cargo.toml]: that sounds quite annoying to make a separate commit for the version bump for every patch but ok. 

Can't you just create a script/github action that changes the version with regex & runs cargo publish?

**Review by conways-glider — APPROVED** [review]
Looks good to me!

**Pull request merged**